megatron/fused_kernels/scaled_upper_triang_masked_softmax.h

```diff
@@ -340,7 +340,6 @@ void dispatch_scaled_upper_triang_masked_softmax_forward(
     int softmax_elements_stride, 
     int attn_batches)
 {
-    TORCH_INTERNAL_ASSERT(softmax_elements >= 0 && softmax_elements <= 2048 );
```

Comment:
I think we should retain these asserts if someone wants to use the cuda code directly.

Reply:
Ok I see. I agree with you.